app/controllers/messages_controller.rb

```diff
@@ -132,4 +141,22 @@ def pusher_message_created(is_single)
       logger.info "PUSHER ERROR: #{e.message}"
     end
   end
+
+  def notify_mentioned_user(user_ids)
+    return unless user_ids
+
+    user_ids.each do |id|
+      message_json = create_pusher_payload(@message, @temp_message_id)
+      Pusher.trigger("private-message-notifications-#{id}", "mentioned", message_json)
+    end
+  end
```

Comment:
I wonder if the Pusher API doesn't have a way to broadcast a message to a group of user, instead of sending possibly many single HTTP calls